integrations/schemas/package-configuration-schema.json

```diff
@@ -143,16 +149,27 @@
   "required": [
     "id"
   ],
-  "oneOf": [
-    {
-      "required": [
-        "vcs"
-      ]
-    },
-    {
-      "required": [
-        "source_artifact_url"
-      ]
-    }
-  ]
+  "not": {
+    "description": "A package configuration must contain at most one of 'vcs', 'source_artifact_url', or 'source_code_origin'.",
```

Comment:
Nit: Isn't it "exactly one of" instead of "at most one of"? And why can't this still use `oneOf`?

Reply:
I'm surprised by that, @nnobelis was it intentional that a package configuration can now contain neither a provenance matcher nor a `sourceCodeOrigin`? I thought the idea was to either provide a provenance matcher or set `sourceCodeOrigin` to some value, but not that this can be left out entirely. There is no explanation in 6d06fccde1169fae40625077e9120a016c3daef7 unfortunately.

Reply:
> Do you mean something like this?

No, I thought "exactly one of" would be correct, but if it's not I don't mind.